### DISCUSSION TITLE
How integrate SKAdNetwork

### DISCUSSION THREAD

**Discussion by JuanFcoRomero** [oddbit/flutter_facebook_app_events#329] [Q&A]
I integrated the plugin and check that events are working. 

But I continue seeing a message of Facebook asking me to update the SDK. 

How can I integrate SKAdNetwork. 

I have instructions for Facebook:

1. Ad this to Podfile. 
-----

Step 1: Update the Facebook SDK for iOS

Via Cocoapods
If your app is integrated with the Facebook SDK for iOS using Cocoapods, follow these steps.

1. Add the code to the Podfile of the packages used in your application.

Example:

pod 'FBSDKCoreKit', '~> 8.0.0'
pod 'FBSDKLoginKit', '~> 8.0.0'
pod 'FBSDKShareKit', '~> 8.0.0'
pod 'FBSDKGamingServiceKit', '~>8.0.0'
2. Run pod update.

But I think the plugin contains that. Right=

The second thing is integrate AdvertiserTrackingEnabled  that I already have. 

How did you do it with the SKAdNework think in iOs?

**Comment by maxIrvine** [comment]
@JuanFcoRomero I am also wondering the same thing. Did you ever find an answer?

**Comment by hilmirstraum** [comment]
Have anyone got an anser for this issue?
